core/arch/arm/kernel/abort.c

```diff
@@ -364,7 +364,10 @@ void abort_print_current_ta(void)
 
 	if (ai.abort_type != ABORT_TYPE_TA_PANIC)
 		__print_abort_info(&ai, "User TA");
-	tee_ta_dump_current();
+
+	EMSG_RAW("Status of TA %pUl (%p)",
```

Comment:
How about moving this text to user space, and drop the context? Arguments "for":
- I don't think the context is useful
- This message logically goes with the subsequent lines (the region dump, the call stack etc.)
- When copying/pasting into `symolize.py`, having a mix of `E/TC` and `E/LD` lines is mildly disturbing (and alignment is not great)
- Anything than can be done in user space should be done there ;-)

For similar reasons, the `arch:` line in `user_ta_dump_state()` would need to move to user space, too.

Reply:
Agree, but this should be done in "core: dump user ta memory map with ldelf"

Reply:
Totally agree ;-)